data-prepper-plugins/s3-source/src/test/java/com/amazon/dataprepper/plugins/source/SqsWorkerTest.java

```diff
@@ -94,6 +127,9 @@ void processSqsMessages_should_return_number_of_messages_processed() {
         assertThat(messagesProcessed, equalTo(1));
         verify(s3Service).addS3Object(any(S3ObjectReference.class));
         verify(sqsClient).deleteMessageBatch(any(DeleteMessageBatchRequest.class));
+        verify(sqsMessagesReceivedCounter).increment(any(Double.class));
```

Comment:
As you are mocking the queue, you should know the actual number here. This is the case for all of the assertions below.

Reply:
How can we get the actual number on mock? Do you mean we know the expected number since we only send 1 message. e.g
```
verify(sqsMessagesReceivedCounter).increment(1);
verify(sqsMessagesDeletedCounter).increment(1);
```